cmd/backup/backup.go

```diff
@@ -42,17 +42,25 @@ func NewBackupCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "backup",
 		Short: "Back-Up k0s configuration. Must be run as root (or with sudo)",
+		PreRun: func(cmd *cobra.Command, args []string) {
+			// ensure logs don't mess up output
+			logrus.SetOutput(cmd.ErrOrStderr())
```

Comment:
We kind of don't. Logrus defaults to os.Stderr. Those additions are mostly safeguards because I was feeling cautious that the config loading stuff might output something new and that could mess up redirects.